tests/Resources/Page/OCPDashboard/OperatorHub/InstallODH.robot

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
+Library   SeleniumLibrary
+
+*** Variables ***
+${DASHBOARD_URL} =    ${OCP_CONSOLE_URL}
```

Comment:
these variables we have already defined so we should use them.

Reply:
I didn't use it in the code.  Completely agree, I forgot to delete this variable. Thanks!